infrastructure/constructs/opentopo.py

```diff
@@ -0,0 +1,25 @@
+from aws_cdk import aws_iam, aws_s3
+from aws_cdk.core import Construct, Tags
+
+from .roles import MAX_SESSION_DURATION
+
+
+class OpenTopography(Construct):
+    def __init__(
+        self, scope: Construct, stack_id: str, *, env_name: str, storage_bucket: aws_s3.Bucket
+    ) -> None:
+        super().__init__(scope, stack_id)
+
+        account_principal = aws_iam.AccountPrincipal(account_id=99)  # TODO: add OT AWS account id
```

Comment:
We probably need to wait until we have the account ID before merging. Don't want to accidentally give some random account access, or fail because that ID doesn't exist.

Reply:
AWS account updated now.